src/time_unit/mod.rs

```diff
@@ -236,7 +237,7 @@ where
         use self::Specifier::*;
         //println!("ordinals_from_specifier for {} => {:?}", Self::name(), specifier);
         match *specifier {
-            All => Ok(Self::supported_ordinals()),
+            All => Ok(Self::supported_ordinals()),  // TODO: change to Self::ALL
```

Comment:
Can you tell me about this `TODO`? Off-hand it seems like `Self::supported_ordinals()` and `Self::ALL.clone()` would do the same work. (The latter might even do more work because `lazy_static` has a small amount of overhead to dereference.)

Reply:
Self::supported ordinals creates a new OrdinalSet. If ALL is already initialized, this should run quicker, otherwise it initializes the OrdinalSet for the next use (instead of throwing the OrdinalSet away)

Reply:
`ALL` is a static ref though, and would need to be `clone()`ed to satisfy the function's return type.

Reply:
That would defeat the purpose then. I can remove the TODO